src/components/ModalSelectDate.vue

```diff
@@ -39,6 +40,17 @@ watch(open, () => {
   time.value = `${h}:${m}`;
   input.value = dateString;
 });
+
+watchEffect(() => {
```

Comment:
Why is this needed now?

Reply:
Okay I see now, but when I enter a time in the past the input is no longer functioning

Reply:
could you specify the way you did it? This code should prevent entering past time.
p.s.: for more clever users I added an extra check [here](https://github.com/snapshot-labs/snapshot/pull/3991/files#diff-2ac25a10c6190e76b7a330fbd6a8438aa8af90245144b2a2e3abba7d5fed8224R162) and [here](https://github.com/snapshot-labs/snapshot-sequencer/pull/71/files#diff-4cd70d1c07b79fdcdf325dd8afe5e8e6af488c09ff1b19a1bf54c1825e349576R105) :)

Reply:
https://www.loom.com/share/e95637017f6c4c9a932fac2fe7a5dda0

Reply:
Instead of filling the input automatically with the current time, we should let the user enter time in that past and clicks "Select" we should show a warning and prevent "Select"